### DISCUSSION TITLE
ReferenceError: Can't find variable: trashEntry

### DISCUSSION THREAD

**Discussion by logifloor** [Laravel-Backpack/community-forum#972] [Q&A (Help)] [answered]
All of a sudden I have started getting this console error when clicking on trash in my list view.  It also does a similar thing when trying to restore and permanently delete.  The bulk operations work fine, and the buttons in the action line are there, they have just stopped doing anything.

Edit and preview are fine, it's just anything related to trash/restore/delete.

I thought it might just be an override on one of the CRUD controllees, but it is affecting CRUDs I haven't modified for months, so it is something site wide I'm experiencing.

Anybody ever had anything similar, or know a good route for bugfixing?

**Comment by logifloor** [comment]
The problem revolves around the @loadOnce directive.  I have removed it temporarily whilst I get to the bottom of why it is not loading the script at all, rather than just loading it once.

**Reply by pxpm** [reply]
I can confirm the issue. Working on a fix ASAP. 🙏 

Thanks for the heads up.

**Comment by karandatwani92** [comment]
Hey @logifloor

I've seen that error that you are talking about here.  Can you run `composer update` and retry?

I tested it on `backpack/pro: 2.2.0` & `backpack/basset: 1.3.3`, and it's working for me.

**Comment by pxpm** [comment, marked as answer]
Hey @logifloor thanks again for the heads up. 

This is now fixed in basset 1.3.4. A `composer update` should get you the fix. Don't forget to clear your caches with `php artisan optimize:clear`

Let me know if you are still experiencing issues. 

Cheers

**Reply by logifloor** [reply]
All working as expected now, thanks 👍
